organization/schema.py

```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
         return QuitKSGMutation(internal_group_position_membership=membership)
 
 
+class InternalGroupUserHighlightNode(DjangoObjectType):
+    class Meta:
+        model = InternalGroupUserHighlight
+        filter_fields = ["internal_group", "user"]
+        interfaces = (Node,)
+
+    image = graphene.String()
+
+    def resolve_image(self: InternalGroupUserHighlight, info, **kwargs):
+        if self.image:
+            return self.image.url
+        else:
+            return None
+
+    @classmethod
+    def get_node(cls, info, id):
```

Comment:
Add a gql login decorator to this

```suggestion
    @classmethod
    @gql_login_required()
    def get_node(cls, info, id):
```

Reply:
done<img src="https://media2.giphy.com/media/26u4lOMA8JKSnL9Uk/giphy.gif"/>